django_stubs_ext/django_stubs_ext/db/models.py

```diff
@@ -0,0 +1,51 @@
+from typing import ClassVar, List, Sequence, Tuple, Union
+
+from django.db.models import BaseConstraint, Index
+from typing_extensions import Literal
+
+from django_stubs_ext import StrOrPromise
+
+
+class BaseModelMeta:
```

Comment:
Perhaps `BaseModelMeta` could be a `Protocol`? Basically just for straining runtime incompatibilities, etc.

```suggestion
class BaseModelMeta(Protocol):
```

Reply:
Haha, never mind, this doesn't work. It'll enforce one to always include all attributes...

Reply:
Ah yes, I was thinking this also, but didn't get to testing it yet. Thanks for trying it out.